### DISCUSSION TITLE
Unable to reproduce AFM magnetic structure

### DISCUSSION THREAD

**Discussion by chtchelkatchev** [CederGroupHub/chgnet#116] [General]
Can not predict AFM magnetic ordering of typical AFMs from Materials Project, like Fe-O mp-1274279, Ni-O, etc... It seems that ChgNet predicts only FM ordering. I have used chgnet.predict_structure and StructOptimizer. Also tried finetuning over my VASP datebase over AFM structures... It did not help.

**Comment by bowen-bd** [comment]
Current version of CHGNet always predict the absolute value of MAGMOMs. So there is no way to distinguish AFM from FM.

**Comment by chtchelkatchev** [comment]
Thank you for quick answer!
